xcm/src/lib.rs

```diff
@@ -55,11 +90,45 @@ pub mod opaque {
 /// A versioned multi-location, a relative location of a cross-consensus system identifier.
 #[derive(Clone, Eq, PartialEq, Encode, Decode, Debug)]
 pub enum VersionedMultiLocation {
-	V0(v0::MultiLocation),
+	V0(v1::MultiLocation),
```

Comment:
why isn't this `V0(Unsupported)` like for `VersionedMultiAsset`

Reply:
Because `MultiLocation` did not change.